BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -13,6 +13,7 @@ import com.braintreepayments.api.IntegrationType.Integration
  * Core Braintree class that handles network requests.
  */
 @Suppress("LargeClass", "LongParameterList", "TooManyFunctions")
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 open class BraintreeClient @VisibleForTesting internal constructor(
```

Comment:
Does this change impact DropIn at all?

Reply:
It shouldn't - drop-in uses the `BraintreeClient` constructor with `BraintreeOptions` param, and has the same "com.braintreepayments.api" group ID as the core libraries, so they all can share these "package-private"/library group methods.